lib/carbon/cache.py

```diff
@@ -116,12 +119,28 @@ def counts(self):
     return [(metric, len(datapoints)) for (metric, datapoints) in self.items()]
 
   @property
+  def unflush_counts(self):
+    return [(metric, self._count_unflushed(datapoints)) for (metric, datapoints) in self.items()]
```

Comment:
can this not be an integer we count? seems unperformant to pay the cost of counting every property call.